pages/common/dolt-version.md

```diff
@@ -0,0 +1,8 @@
+# dolt version
+
+> Displays the current Dolt cli version.
+> More information: <http://github.com/dolthub/dolt>.
+
+- Display the current Dolt cli version:
```

Comment:
```suggestion
- Display `dolt` version:
```
Or
```suggestion
- Display version:
```
I think the command description is self-explanatory so this would be better.

Reply:
solved in 74cec9a7f6adbce48e3678548f1f10b58aad33eb